github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -266,3 +266,52 @@ periodics:
         resources:
           requests:
             memory: "29Gi"
+- name: periodic-kubevirt-mirror-uploader
+  cluster: ibm-prow-jobs
+  cron: "05 7 * * *"
+  max_concurrency: 1
+  decorate: true
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  extra_refs:
+  - org: fgimenez
+    repo: project-infra
+    base_ref: add-git-pr-script
+    workdir: true
+  - org: kubevirt
+    repo: kubevirt
+    base_ref: master
+  spec:
+    containers:
+      - image: "kubevirt/builder@sha256:e6a76ec71f2de87c3bc648bae87ab43a113540bf57da69f07f76bd0edfd500ab"
+        env:
+        - name: GIT_ASKPASS
+          value: "hack/git-askpass.sh"
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /etc/gcs/service-account.json
+        command: ["/bin/sh"]
+        args:
+          - "-c"
+          - curl -L https://golang.org/dl/go1.15.3.linux-amd64.tar.gz --output go1.15.3.linux-amd64.tar.gz &&
```

Comment:
That is a nice idea. in general we have golang already in there.  The issue is that the environment is not initialized properly, since go was installed via gimme.